### PR TITLE
[Php81] Handle nested Fqcn UniqueConstraint on NestedAnnotationToAttributeRector

### PR DESCRIPTION
**Pull request by TomasVotruba**
- cs
- add failng test case

### CODE REVIEW

File: src/PhpAttribute/NodeFactory/PhpNestedAttributeGroupFactory.php [hunk 1]
```diff
@@ -212,6 +212,7 @@ private function createFromExplicitProperties(
 
             foreach ($nestedArrayItemNode->value->getValues() as $arrayItemNode) {
                 $nestedDoctrineAnnotationTagValueNode = $arrayItemNode->value;
+
                 if (! $nestedDoctrineAnnotationTagValueNode instanceof DoctrineAnnotationTagValueNode) {
                     throw new ShouldNotHappenException();
```
Comment (TomasVotruba) [inline src/PhpAttribute/NodeFactory/PhpNestedAttributeGroupFactory.php]: Seems this line is breaking

**Comment by samsonasik** [comment]
Fixed 🎉

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba let's merge it ;)

**Comment by TomasVotruba** [comment]
Wow, great job with figuring this out  :pray:

**Pull request merged**